share/spice/cricket/cricket.css

```diff
@@ -0,0 +1,136 @@
+.zci--cricket .card {
+    background: #FFFFFF;
+    font-size: 16px;
+    box-shadow: 0 10px 20px rgba(0, 0, 0, 0.19), 0 6px 6px rgba(0, 0, 0, 0.23);
+    box-sizing: border-box;
+}
+
+.zci-cricket-content {
+    color: #888;
+    padding: 20px;
+    border-top: 2px solid #F7F7F7;
+    border-bottom: 2px solid #F7F7F7;
+}
+
+.zci-cricket-content > span:first-child {
+    float: left;
+}
+
+.zci-cricket-content > span:last-child {
+    float: right;
+}
+
+.zci-cricket-team {
+    display: inline-block;
+    padding-left: 20px;
+    padding-right: 20px;
+    min-width: 200px;
+}
+
+.zci-cricket-team > span {
+    display: block;
+}
+
+.zci-cricket-team > span > span > span {
+    float: right;
+    padding-left: 40px;
+}
+
+.zci-cricket-team-expanded {
+    padding-top: 20px;
+    width: 100%;
+}
+
+.zci-cricket-team-expanded > div {
+    display: inline-block;
+
+}
+
+.zci-cricket-team-expanded > div:first-child {
+    /*float: left;*/
+}
+
+.zci-cricket-team-expanded > div:last-child {
+    float: right;
+    padding-right: 20px;
+}
+
+.zci-cricket-match-result {
+    text-align: center;
+}
+
+.zci-cricket-team-expanded-logo {
+    vertical-align: middle;
+    max-width: 100px;
+    max-height: 50px;
+}
+
+.zci-cricket-inactive {
+    display: none;
+}
+
+.zci-cricket-active {
+    display: block;
+}
+
+.zci-cricket-score-head {
+    color: #888;
+    padding: 20px;
+    padding-bottom: 40px;
+}
+
+.zci-cricket-score-content {
+    color: #888;
+    padding: 20px;
+    border-top: 2px solid #F7F7F7;
+}
+
+.zci-cricket-score-head-date {
+
+}
+
+.zci-cricket-team-logo {
+    vertical-align: middle;
+    height: 21px;
+    float: left;
+    max-width: 29px;
+}
+
+.zci-cricket-team-name {
+    /*float: left;*/
+}
+
+.zci-cricket-match {
+    color: #888;
+    padding: 20px;
+    border-bottom: 2px solid #F7F7F7;
+}
+
+.chevron::before {
+    border-style: solid;
+    border-width: 0.25em 0.25em 0 0;
+    content: '';
+    display: inline-block;
+    height: 0.45em;
+    left: 0.15em;
+    position: relative;
+    top: 0.15em;
+    vertical-align: top;
+    width: 0.50em;
+    color: #D18B49;
```

Comment:
All text colours should be applied using our classes, e.g. `text-primary` etc -- this ensure the colors are correct when the user switches themes

Checkout https://duckduckgo.com/styleguide